### ISSUE TITLE
Provide PNG artwork

### ISSUE THREAD

**Issue by wu-lee** [TransitionbyDesign/homemaker#18]

**Comment by TransitionbyDesign** [comment]
- I've managed to get the first content type - image with text - ready in the structure plan.
[The DIY House (Notion)](https://www.notion.so/transitionbydesign/DIY-House-image-60d9f961399747049a97160174d4407e)

- Jas is now going to create a few alternatives for the 'story pages' mock up include a few images for the DIY house. Idea would be to have a gallery function in Forestry but we will send you what that looks like as soon as he's done the work early next week.
 
- Lucy and the team are sprinting to create more of the story pages in the next fortnight. They essentially all look the same but have an embedded video or audio track instead (or alongside) an image

**Comment by wu-lee** [comment]
Thanks. 

Would it be possible to put the text raw content somewhere? i.e. Text as you intend to supply it, whether that be HTML, Markdown, or something else. Attaching it here would do, for instance.

Will the text contain internal links?

Can I also clarify if that is literally "[an] image with text", or could it be "[N] images with text"?  Or perhaps the gallery function would provide some small number of alternatives in terms of images (which might require some work to facilitate)?

**Comment by TransitionbyDesign** [comment]
Yeah I can put the markdown text here if  you want and I'll start collating
a set of folders of images to go with the text. You should be able to copy
the text out of the notion page as markdown - remember this is just draft
for creating one of the four content types.

The entire notion text structure should export as markdown as well which is
useful as Notion is definitely the easiest way of me gathering content from
the team.

There will be multiple images for some stories but Jas is just having a
think about how that could work and will get back to us early this week.

**Comment by Charlie-TbD** [comment]
Morning

As agreed I'm continuting to set the [structure up in Notion](https://www.notion.so/transitionbydesign/Homemaker-Website-Content-c98c4ae439304303be3db532d46d9fd4). I've got a few content types with a little bit of text to go with it. 

[Draft content pages ready for website
](https://www.notion.so/transitionbydesign/b882574808d246d59403e6bb5a01592b?v=8d7335d8ace149fc90bb544e43d38aac)

The most important one will be a holding page which is the same as the landing pages but missing the links. This is in case people aren't on the proejct team stumble on the website. I'll add this as a separate issue.

**Questions**
- [ ] What format do you want the pin location putting in? Lat/Long?

**Comment by wu-lee** [comment]
Pin locations: yes, please use latitude/longitude.

**Comment by Charlie-TbD** [comment]
I think this is superceded now so will move to done
